settings.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import unicode_literals
+import os
+import dj_database_url
+
+# import our default settings
+from casepro.settings_common import * # noqa
+
+import environ
+
+env = environ.Env(DEBUG=(bool, False))
+# reading .env file
+environ.Env.read_env()
+
+DEBUG = False
+SECRET_KEY = env.str("SECRET_KEY", default='REPLACEME')
+TEMPLATE_DEBUG = DEBUG
+COMPRESS_OFFLINE = True
+SEND_EMAILS = True
+
+HOSTNAME = env.str('HOSTNAME', default='localhost:8000')
+SITE_HOST_PATTERN = env.str('SITE_HOST_PATTERN', default='http://%s.localhost:8000')
+
+SITE_API_HOST = env.url('SITE_API_HOST', default='http://localhost:8001/')
```

Comment:
I think this gives us an `urlparse.ParseResult`, not a string, and I think this field is supposed to be a string.